services/director-v2/src/simcore_service_director_v2/modules/comp_scheduler/__init__.py

```diff
@@ -1,3 +1,15 @@
-from .background_task import setup
+from fastapi import FastAPI
 
-__all__: tuple[str, ...] = ("setup",)
+from ._base_scheduler import BaseCompScheduler
+from ._task import on_app_shutdown, on_app_startup
+
+
+def setup(app: FastAPI):
+    app.add_event_handler("startup", on_app_startup(app))
```

Comment:
THOUGHT: we need to start seriously exploring and moving to [lifespan-events](https://fastapi.tiangolo.com/advanced/events/#lifespan-events)

Reply:
yes, but this did not change in this PR.